qemu/tests/steal_time.py

```diff
@@ -0,0 +1,72 @@
+import logging
+import time
+import re
+
+from virttest import error_context
+from virttest import utils_test
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Steal time test:
+    1) Boot two guests and bind to one same cpu
+    2) Run stress inside both guests
+    3) Check steal time in top inside guests
+    4) Check steal time in /proc/stat in guest
```

Comment:
No need to check the qemu-kvm cpu usage in host ?

Reply:
Yes, I may have missed this step, updated, pls help review again, thanks.